packages/SwingSet/demo/encouragementBot/bootstrap.js

```diff
@@ -1,13 +1,14 @@
 import { E } from '@agoric/eventual-send';
 import { Far } from '@endo/marshal';
 
-console.log(`=> loading bootstrap.js`);
+console.log('=> loading bootstrap.js');
 
 export function buildRootObject(vatPowers) {
   const log = vatPowers.testLog;
+  log('=> buildRootObject called');
```

Comment:
I'm confused by this delta because `vatPowers.testLog` is not referenced in the master branch version of this file, but line 7 doesn't show up as a diff -- it shows the "before" file as having it also.  So I'm not sure what this PR is a PR against.  So something is wrong here.

In any case, you can't use testLog in a swingset-runner demo.  Well, you can, but it's a bit bucket and there will be no output.

Reply:
I somehow got confused seeing `demo` in the path and encoded that this was the `swingset-runner` version, not the `SwingSet` version.  I feel better now!